src/views/common/Page404.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <div class="main-slide-fade-container">
+
+    <div class="x">
+
+      <h2 class="mb">Тут ничего нет</h2>
+      <p class="mb">То, что ты ищешь, в настоящее время не существует по этому адресу.</p>
+      <p class="mb">Если только ты не ищешь страницу с ошибкой, в таком случае: поздравляю!</p>
+      <p>Ты нашел её 😘</p>
+    </div>
+
+  </div>
+
+</template>
+
+<script>
+  export default {
+    name: "Page404",
+    mounted() {
+      setTimeout(this.goHome, 3500)
```

Comment:
I can’t understand if forced redirection is needed here, without any notifications it will be incorrect to redirect users, you need to display a timer or make the button “return to the main”, it will be more intuitive.

Reply:
Interesting idea - timer or interactive element. added to Ideas